models/br_bd_execucao_estadual/code/register_coverage.py

```
@@ -0,0 +1,201 @@
+        month = MONTH_COLUMN.get(slug)
+        extra = (
+            f", min(ano * 100 + {month}) ym0, max(ano * 100 + {month}) ym1"
+            if month
+            else ""
+        )
+        try:
+            rows = client.query(
+                f"select sigla_uf, min(ano) y0, max(ano) y1{extra} "
+                f"from `{project}.br_bd_execucao_estadual.{slug}` group by sigla_uf"
+            ).result()
+        except Exception as exc:  # table not built yet
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)